frontend/src/pages/device/Logs.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <SectionTopMenu hero="Device Logs" help-header="FlowForge - Device Logs" info="Live logs from your FlowForge instances of Node-RED">
+        <!-- <template>
+        </template>
+        <template #tools>
+        </template> -->
```

Comment:
```suggestion
```

Should remove this?

Reply:
last one @hardillb 

Is it necessary to keep commented code?

Reply:
I don't think so, I removed them when talking to Pez about the lint changes. I'll commit this suggestion.

Please pull and re-test this before merging just to be sure